cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala

```diff
@@ -206,6 +207,43 @@ class SchemaIndexTest extends DocumentingTestBase with QueryStatisticsTestSuppor
     )
   }
 
+  @Test def create_text_index() {
+    val textProvider = TextIndexProviderFactory.DESCRIPTOR.name()
+
+    testQuery(
+      title = "Create a node text index",
+      text = "A named text index on a single property for all nodes that have a particular label can be created with `CREATE TEXT INDEX index_name FOR (n:Label) ON (n.property)`. " +
+        "Note that the index is not immediately available, but is created in the background.",
+      queryText = "CREATE TEXT INDEX node_index_name FOR (n:Person) ON (n.surname)",
+      optionalResultExplanation = "Note that text indexes do not support multiple properties and that the index name needs to be unique.",
+      assertions = _ => assertIndexWithNameExists("node_index_name", "Person", List("surname"))
+    )
+    testQuery(
+      title = "Create a relationship text index",
+      text = "A named text index on a single property for all relationships that have a particular relationship type can be created with `CREATE TEXT INDEX index_name FOR ()-[r:TYPE]-() ON (r.property)`. " +
+        "Note that the index is not immediately available, but is created in the background.",
+      queryText = "CREATE TEXT INDEX rel_index_name FOR ()-[r:KNOWS]-() ON (r.interest)",
+      optionalResultExplanation = "Note that text indexes do not support multiple properties and that the index name needs to be unique.",
+      assertions = _ => assertIndexWithNameExists("rel_index_name", "KNOWS", List("interest"))
+    )
+    testQuery(
+      title = "Create a text index only if it does not already exist",
+      text = "If it is unknown if an index exists or not but we want to make sure it does, we add `IF NOT EXISTS`.",
```

Comment:
```suggestion
      text = "If it is not known whether an index exists or not but to ensure it does, add `IF NOT EXISTS`.",
```

Reply:
Your sentence sounds odd to me, `... exists or not but to ensure it does ...` feels weird

Reply:
I rewrote it someway lets see what you think